<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBlock</class>
 <widget class="QDialog" name="DialogBlock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_block">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Block</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_ip">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QListView" name="lv_ip">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>611</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_unblock">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Unblock</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
